### ISSUE TITLE
[Feature Request] Send Bed and Cabinet to Storage

### ISSUE THREAD

**Issue by Sourceshard** [d-rafferty/lethal-tube-removal#16]
Would it be possible to send the Bed and the Cabinet to storage instead of deleting them entirely? I ask because I remove them for a mod pack I am making. However, I know that there are popular mods that utilize these items for their mechanics, and a player might want to add these mods into my modpack. I would like for my changes to the ship to be as compatible as possible.

Thanks in advance. :D

**Comment by d-rafferty** [comment]
> Would it be possible to send the Bed and the Cabinet to storage instead of deleting them entirely? 

Of course! I'll work on this and push an update out later today.

**Comment by d-rafferty** [comment]
A bit more annoying than initially expected to get the game to allow me to automate the process of sending something to storage through build mode. Still working on it though.

**Comment by d-rafferty** [comment]
I still dabble with this issue here and there, but thus far I have not been able to get anything to work. I am a Software Engineer who mostly works on embedded systems so everything I know about Unity code or modding in general has came from making this mod. Needless to say I know more than the layman but I honestly have no idea what I'm doing and I just wing it. I don't have much time to spend on this mod these days but maybe eventually I'll figure it out or someone else will happen upon this issue and know what to do.. lol

TLDR: sorry for saying I'd get that done quick because I thought I could do it quickly, when I in fact; could not
